asset-transfer-basic/chaincode-external/assetTransfer.go

```diff
@@ -0,0 +1,219 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+type ServerConfig struct {
+	CCID    string
+	Address string
+}
+
+// SmartContract provides functions for managing a car
+type SmartContract struct {
+	contractapi.Contract
+}
+
+// Asset describes basic details of what makes up a simple asset
+type Asset struct {
+	ID             string `json:"ID"`
+	Color          string `json:"color"`
+	Size           int    `json:"size"`
+	Owner          string `json:"owner"`
+	AppraisedValue int    `json:"appraisedValue"`
+}
+
+// QueryResult structure used for handling result of query
+type QueryResult struct {
```

Comment:
No need to export

Reply:
Same as `Asset`, `QueryResult` is being returned by `GetAllAssets`